li-utils/src/main/pegasus/com/linkedin/common/DataAccessUrn.pdl

```diff
@@ -0,0 +1,27 @@
+namespace com.linkedin.common
+
+/**
+ * Standardized data access identifier.
+ */
+@java.class = "com.linkedin.common.urn.DataAccessUrn"
+@validate.`com.linkedin.common.validator.TypedUrnValidator` = {
+  "accessible" : true,
+  "owningTeam" : "urn:li:internalTeam:datahub",
+  "entityType" : "dataAccess",
+  "constructable" : true,
+  "namespace" : "li",
+  "name" : "DataAccess",
+  "doc" : "Identifier for data access",
+  "owners" : [ "urn:li:corpuser:fbar", "urn:li:corpuser:bfoo" ],
+  "fields" : [ {
+    "type" : "com.linkedin.common.urn.DatasetUrn",
```

Comment:
tying the "key" of the data access request too closely with the kind of entity you are applying for has the problem of making it hard to generalizing this to other kinds of entities. So I would relax this requirement.

Reply:
Could you provide examples of other types of entities where the access would be granted? I understand that the dataset is ultimately the final object at the end of the chain where access must be granted. However, I might be mistaken. Could you clarify?

Reply:
If we consider a Dashboard object as an example, it encompasses Charts, which in turn contain Datasets. It is the Datasets where access ultimately needs to be granted, unless we are also responsible for providing access to the Charts and Dashboard respectively. But this is just an example, maybe you could give us another useful examples of another entities.

Reply:
Thanks @gamboad for the comments. Another comment from my side:

Although keeping generalization in mind, we put dataset URN here according to the name `DataAccess` and the fact that its aspect DataAccessProperties contains attributes specific to datasets (e.g `fieldAccesses`) but not available with other types of assets.

If we consider other types of assets, I wonder it might make sense to introduce a general new entity like `AssetAccess`, `ContentAccess` or similar, which could have different aspects, and they can be one level higher than `DataAccess`. As an example: One instance of `ContainerAccess` might contain a list of `DataAccess` instances in its aspect attribute, which could also reflect the permission structure in the data platform (e.g. in GCP there is dataset/schema level permission and table/view level permission).

Reply:
> could you provide other examples of entities that require access permissions?

Access may be requested at container level (_eg_ redshift database) or even at platform instance level.